src/content/staking/withdrawals/index.md

```diff
@@ -0,0 +1,170 @@
+---
+title: Staking withdrawals
+description: Page summarizing what staking push withdrawals are, how they work, and what stakers need to do to get their rewards
+lang: en
+template: staking
+image: ../../../assets/staking/leslie-withdrawal.png
+alt: Leslie the rhino with her staking rewards
+sidebarDepth: 2
+summaryPoints:
+  - The Shanghai upgrade brings staking withdrawals to Ethereum
+  - Validator operators must provide a withdrawal address to enable
+  - Rewards payments are automatically sent to the address provided every few days
```

Comment:
```suggestion
  - Reward payments are automatically sent to the address provided every few days
```
I think just "reward payments"? The double plural feels weird (but I'm no grammar expert)

Reply:
```suggestion
  - Rewards are automatically sent to the address provided every few days
```
Maybe the "payments" is not needed?

Reply:
Advocating for "payments" as it distinguishes between just your earned rewards, and the act of moving those rewards in the form of a payment.

Given that these are "rewards", not "a single reward", and that we are talking about the payouts in a pluralistic way, as in "payments", that's why "rewards payments" was chosen here... But I get it... "Reward payments" is fine with me if people think it's too awkward